js/index.js

```diff
@@ -0,0 +1,22 @@
+$(function(){
+
+$('#stopButton').click(switchRed);
+
+function switchRed(){
+	$('#stopLight').css('background-color','red');
+}
+
+$('#slowButton').click(switchYellow);
+
+function switchYellow(){
+	$('#slowLight').css('background-color','yellow')
+}
+
+$('#goButton').click(switchGreen);
+
+function switchGreen(){
+	$('#goLight').css('background-color','green')
+}
+
```

Comment:
@kelseycoughlin you forgot to make the `clearLights` function. You would then have to call that function each time you call one of your switch color functions. That way it will make the color black first before changing the color 🔨